code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -241,6 +241,14 @@ Difficulty: Very Hard
 		AT.pixel_y += random_y
 	return ..()
 
+/mob/living/simple_animal/hostile/megafauna/colossus/float(on) //we don't want this guy to float, messes up his animations
```

Comment:
If we dont want the colossus to float, why not just override this to `return` with nothing else, or am I missing something.

Reply:
Done for recording keeping purposes regarding `floating`.